src/common/transformations/include/transformations/rt_info/reduceop_path.hpp

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/core/node.hpp"
+#include "openvino/core/runtime_attribute.hpp"
+#include "transformations_visibility.hpp"
+
+namespace ov {
+
+TRANSFORMATIONS_API void mark_reduceop_path(const std::shared_ptr<Node>& node);
```

Comment:
Do we really need to have these methods as a part of API? I see that it is used only in `src/common/transformations/src/transformations/common_optimizations/mark_subgraphs_to_keep_in_mixed_precision.cpp`... In this case these methods can be internal.

Moreover I don't see that you remove this flag from the rt info after the transformation pipeline. We shouldn't keep garbage in the rt info after transformations.

Reply:
communicated personally with Ilya, i will move this rt_info from public API into transformation cpp

Reply:
Moved `reduceop_path` rt_info attr inside cpp file